garrysmod/gamemodes/terrortown/gamemode/gamemsg.lua

```diff
@@ -245,9 +249,13 @@ local function MuteTeam(ply, cmd, args)
 
    local t = tonumber(args[1])
    ply.mute_team = t
-
-   local name = (t != 0) and team.GetName(t) or "None"
-   ply:ChatPrint(name .. " muted.")
+   if t == MUTE_ALL then
+   ply:ChatPrint("All muted.")
+   elseif t == MUTE_NONE or t == TEAM_UNASSIGNED then

```

Comment:
Is there any way to specify that only this numbers can be used?
For example if you use "ttt_mute_team 60" you will get printed " muted.".
This would work:
`   if t != MUTE_ALL and t != MUTE_NONE and t != TEAM_UNASSIGNED and t != MUTE_TERROR and t != MUTE_SPEC then
   ply.mute_team = -1
   ply:ChatPrint("None muted.")
   else`
But is there anything that look better?